doc/quickstart/ncWMS_quickstart.rst

```diff
@@ -78,21 +81,25 @@ Certain variables, particularly biological parameters, are best displayed with a
 Creating animations
 -------------------
 
-1) Click the animation wizard button below the lower left corner of the map.
-2) Select a start and end time for the animation
-3) Choose the granularity and frame rate for the animation
-4) Click "Go". Note that the animation may take quite a while to appear.
-5) Click the "Stop animation" button (where the wizard was) to stop the animation and return the controls to normal.
+#. Click the animation wizard button below the lower left corner of the map.
```

Comment:
The whole document could be improved by the paragraph sections being numbered steps like this.